recipes/usrsctp/all/conandata.yml

```diff
@@ -0,0 +1,3 @@
+sources:
+  "master":
```

Comment:
Use official releases only. If it's too old, ask to the upstream for a new one. When abandoned, you can create a version based on the commit date: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases